src/Plugins/NamedPipeService/Models/Payloads/PipeNullPayload.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2015-2024 The Neo Project.
+//
+// PipeNullPayload.cs file belongs to the neo project and is free
+// software distributed under the MIT software license, see the
+// accompanying file LICENSE in the main directory of the
+// repository or http://www.opensource.org/licenses/mit-license.php
+// for more details.
+//
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+using System.IO;
+
+namespace Neo.Plugins.Models.Payloads
+{
+    internal sealed class PipeNullPayload : IPipeMessage
```

Comment:
Why we need it? maybe ping, but null?

Reply:
If you request a `Block` for example. If may not be at the that height yet or doesn't exists _(if you request something else)_. In this case we return `null`

Reply:
In short, just for empty protocol messages. Like command `GetBlockHeight`